tutormfe/templates/mfe/build/mfe/Dockerfile

```diff
@@ -78,6 +78,8 @@ ENV PUBLIC_PATH='/{{ app_name }}/'
 # So we point to a relative url that will be a proxy for the LMS.
 ENV MFE_CONFIG_API_URL=/api/mfe_config/v1
 ARG ENABLE_NEW_RELIC=false
+{{ patch("mfe-dockerfile-pre-npm-build") }}
+{{ patch("mfe-dockerfile-pre-npm-build-{}".format(app_name)) }}
```

Comment:
Could you please document these new patches in the README?

Reply:
Done